packages/codeeditor/src/lineCol.tsx

```diff
@@ -107,6 +107,7 @@ class LineFormComponent extends React.Component<
             <input
               type="text"
               className="jp-lineFormInput"
+              aria-label={this._trans('Line Column form')}
```

Comment:
I think it would be better to give this text input an id and connect it to the label below, e.g.:

```
const textInputId = DOMUtils.createDomID() + '-line-number-input';
<input id={textInputId} ... />
...
<label for={textInputId}
```

Reply:
This would be `htmlFor={textInputId}` because React.